gatsby-node.js

```diff
@@ -61,6 +61,36 @@ exports.onCreateNode = async ({node, actions, loadNodeContent, createNodeId, cre
             const obj = YAML.parse(content);
             obj.slug = path.basename(node.dir);
 
+            // Move fields from map to metadata
+            if (obj.map) {
+                if (!obj.metadata) {
+                    obj.metadata = {};
+                }
+
+                const mapFields = ['name', 'submitted_by', 'authored_by', 'location', 'industries', 'latitude', 'longitude'];
```

Comment:
Regarding
```
                const mapFields = ['name', 'submitted_by', 'authored_by', 'location', 'industries', 'latitude', 'longitude'];
```
Why do we still have the "name" and "submitted_by" fields in this PR?

Reply:
Hi Sir,

Thank you for pointing that out! The presence of the name and submitted_by fields in the PR is unintentional. I initially retained these fields during the migration to ensure no data was lost, but I missed cleaning them up in the final implementation.

I will update the PR to remove these redundant fields and ensure that only the authored_by field is used consistently across the codebase. 

Thank you for catching this, and I’ll notify you as soon as the changes are made.

Reply:
I have made the changes and also tested it locally to check if everything runs fine !

<img width="1512" alt="Screenshot 2025-01-01 at 1 25 26 AM" src="https://github.com/user-attachments/assets/83957889-5cd0-44c2-be41-5f0db2dd296b" />

<img width="1512" alt="Screenshot 2025-01-01 at 1 25 44 AM" src="https://github.com/user-attachments/assets/1a14ee9c-a5fc-445f-a228-38ba4bf99bbb" />

<img width="1512" alt="Screenshot 2025-01-01 at 1 26 07 AM" src="https://github.com/user-attachments/assets/b3de4d5b-d0db-44be-9c11-bcc990b555c4" />

<img width="1512" alt="Screenshot 2025-01-01 at 1 28 52 AM" src="https://github.com/user-attachments/assets/cdb87ed4-a15f-433b-910d-903081947452" />